mm/2s2h/GameInteractor/GameInteractor_VanillaBehavior.h

```diff
@@ -1793,6 +1793,25 @@ typedef enum {
     // - `*s32` (which boss remains to return, see `func_808B849C`)
     VB_SPAWN_BOSS_REMAINS,
 
+    // #### `result`
+    // ```c
+    // true
+    // ```
+    // #### `args`
+    // - None
+    VB_SPEED_MODIFIER_WALK,
+
+    // #### `result`
+    // ```c
+    // true
+    // ```
+    // #### `args`
+    // - '*f32' (incrStep)
+    // - '*f32' (maxSpeed)
+    // - '*f32' (speed)
+    // - '*f32' (speedTarget)
+    VB_SPEED_MODIFIER_SWIM,
```

Comment:
Swap these so they're alphabetical, and we're golden.

```suggestion
    // #### `result`
    // ```c
    // true
    // ```
    // #### `args`
    // - '*f32' (incrStep)
    // - '*f32' (maxSpeed)
    // - '*f32' (speed)
    // - '*f32' (speedTarget)
    VB_SPEED_MODIFIER_SWIM,

    // #### `result`
    // ```c
    // true
    // ```
    // #### `args`
    // - None
    VB_SPEED_MODIFIER_WALK,
```